chronosphere/intschema/shared_schemas.go

```diff
@@ -11,6 +11,19 @@ type LogAllocationConfigSchema struct {
 	PercentOfLicense float64 `intschema:"percent_of_license"`
 }
 
+type LogPrioritiesSchema struct {
+	HighPriorityFilters []LogPrioritiesSchemaHighPriorityFilters `intschema:"high_priority_filters,optional"`
+	LowPriorityFilters  []LogPrioritiesSchemaLowPriorityFilters  `intschema:"low_priority_filters,optional"`
+}
+
+type LogPrioritiesSchemaHighPriorityFilters struct {
+	Query string `intschema:"query"`
+}
+
+type LogPrioritiesSchemaLowPriorityFilters struct {
+	Query string `intschema:"query"`
+}
```

Comment:
these 2 are the same type, we should have a shared type and shared conversion functions